From 9857c80d9dfa8c635ab33d17bcd54e058d3a0d56 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Mon, 11 Mar 2019 16:39:28 +0800
Subject: [PATCH] ARM: dts: rockchip: rk312x-android: set vop-dclk-mode default
 value to 1

Fix display abnormal caused by DDR frequency conversion.

Change-Id: Iaa3bf6177d42f8ac5f9078b58a138f48d5c1d874
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-android.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk312x-android.dtsi b/arch/arm/boot/dts/rk312x-android.dtsi
index f676c820edf3..77736bb915ad 100644
--- a/arch/arm/boot/dts/rk312x-android.dtsi
+++ b/arch/arm/boot/dts/rk312x-android.dtsi
@@ -105,6 +105,7 @@
 };
 
 &dmc {
+	vop-dclk-mode = <1>;
 	status = "okay";
 };
 
-- 
2.35.3

